tests/rbd/rbd_utils.py

```diff
@@ -773,6 +773,35 @@ def set_config(self, level, entity, key, value, **kw):
         """
         return self.exec_cmd(cmd=f"rbd config {level} set {entity} {key} {value}", **kw)
 
+    def set_ceph_config(self, entity, key, value, **kw):
+        """Override ceph config settings.
+
+        ceph config set <entity> <key> <value>
+
+        Args:
+            entity: config entity (mon, osd)
+            key: config option key (ex., rbd_move_to_trash_on_remove)
+            value: config option value to be set
+            kw: other ceph arguments like config options
+        Returns:
+            exec_cmd response
+        """
+        return self.exec_cmd(cmd=f"ceph config set {entity} {key} {value}", **kw)
+
+    def get_ceph_config(self, entity, key, **kw):
+        """Get ceph config settings.
+
+        ceph config get <entity> <key>
+
+        Args:
+            entity: config entity (mon, osd)
+            key: config option key (ex., rbd_move_to_trash_on_remove)
+            kw: other ceph arguments like config options
+        Returns:
+            exec_cmd response
+        """
+        return self.exec_cmd(cmd=f"ceph config get {entity} {key}", **kw)
```

Comment:
Same, please move this method out of the RBD class.

Reply:
Done